### PR TITLE
#5 Added new header component "machine_site_uuid" inside authenticateAsync-method

### LINKED ISSUE #5: Add a new header component "msupply_site_uuid" to track device unique id
Need to add a new header component "msupply_site_uuid" to track device unique id in the `authenticateAsync` method

e.g. 
screenshot 2018-12-25 at 09 56 52

**Refer**
[issue](https://github.com/sussol/mobile/issues/762) for more details

### PR DESCRIPTION
**Pull request by raviSussol**
Fix #5 

**Description**
Added a new header component `machine_site_uuid` inside `authenticationAsync` method to track/check device unique id.

**Refer**
[issue](https://github.com/sussol/mobile/issues/762) for more details

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -20,9 +20,10 @@ const {
 * @param  {string}   authURL   The URL to authenticate against
 * @param  {string}   username  The username to test
 * @param  {string}   password  The password to test
+* @param  {string}   machineUUID The machine UUID to check - optional
 * @return {object}             JSON formatted response object
 */
-export async function authenticateAsync(authURL, username, password) {
+export async function authenticateAsync(authURL, username, password, machineUUID = '') {
```
Comment (andreievg) [inline src/index.js]: @raviSussol -> can the last parameter be a little bit more dynamic, i suggest making it a JSON.

i.e.
```
export async function authenticateAsync(authURL, username, password, extraHeaders = {})
...
   Authorization: getAuthHeader(username, password),
   ...extraHeaders  
```

**Comment by andreievg** [comment]
@raviSussol, another adjustment to this PR: don't need to commit both lock files (no new dependencies added)

**Comment by andreievg** [comment]
@raviSussol, I am not sure how those lock files would affect projects where authentication-utilities is already use. I still think that you should remove them from this PR. If you want to update yarn.lock and package-lock.json for future project, we can make a seperate issue for that.

**Comment by raviSussol** [comment]
> @raviSussol, I am not sure how those lock files would affect projects where authentication-utilities is already use. I still think that you should remove them from this PR. If you want to update yarn.lock and package-lock.json for future project, we can make a seperate issue for that.

@andreievg, After battling with the Git, finally removed 2 lock files. 😄

**Comment by andreievg** [comment]
@raviSussol, thanks, changes look good.
btw, instead of reverting commits. it's often better to replace the state of file with previous version and re-commit (i.e. in this case, remove yarn lock and npm lock and re-commit removal)

**Pull request merged**